iOS/SonarKit/CppBridge/SonarCppBridgingResponder.mm

```diff
@@ -7,7 +7,7 @@
  */
 #import "SonarCppBridgingResponder.h"
 
-#import <FBCxxUtils/FBCxxFollyDynamicConvert.h>
```

Comment:
This changes will break internally for us. You can either give the path like `Pods/SonarKit/iOS/SonarKit` or create a podspec for the same. I think creating podscpec for FBCxxUtils will be the good way around.